src/alchemtest/gmx/access.py

```diff
@@ -102,3 +102,72 @@ def load_expanded_ensemble_case_3():
 
     return Bunch(data=data,
                  DESCR=fdescr)
+
+def load_water_particle_without_energy():
+    """Load the Gromacs water particle without energy dataset.
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files by alchemical leg
+        - 'DESCR': the full description of the dataset
+
+    """
+
+    module_path = dirname(__file__)
+
+    data = {'AllStates': glob(join(module_path, 'water_particle/without_energy/*.xvg.bz2'))}
```

Comment:
Don't use Unix path separators but let `join` do everything:
```python
glob(join(module_path, 'water_particle', 'without_energy', '*.xvg.bz2'))
```
to make sure it also works on Windows.

Reply:
Oh I wasn't aware of that. I guess this should also be fixed in the other accessors functions.

I'll create a separate PR for that.

Reply:
Yes, I am sure. Sorry, sometimes the new PR is the one that reveals a general shortcoming in old code...